### ISSUE TITLE
Change Request: Medication Wording for Appointments (OIG/Congressional request)

### ISSUE THREAD

**Issue by kristenmcconnell** [department-of-veterans-affairs/va-mobile-app#9373]
### 1.) Requestor's Name

Kristen McConnell

### 2.) Requestor's Email

kristen.mcconnell@va.gov

### 3.) VA Team

Unified Appointment Team

### 4.) What is the name of the project or change?

Medication Wording for Appointments (OIG/Congressional request)

### 5.) Please explain what problem you are trying to solve.

The objective is to remind Veterans to be ready to talk about any medication, supplement, vitamins, etc that they have been prescribed or are taking from any provider, VA clinic, or local store.

### 6.) Please explain how this change would work.

The plan is to add wording to the appointment detail pages. [Mobile wireframes](https://www.figma.com/design/ZIHY63cFlD7pFjqQLA5rFN/Appointments-2.0---%F0%9F%94%8D-Working---VA-Mobile?node-id=3169-3473&t=j3Rv3NeisvIsR7Mi-1) were already started as a joint effort between the Unified Appointment and Mobile teams. 

This is proposed to be the first engineering task for the Unified Appointments new mobile engineer to take on, so the ask for the mobile team is to use this intake to help set up any tickets they need to help the UAE engineer through the process this first time around.

### 7.) Features Impacted

Appointments

### 8.) Please include all relevant documentation links for this change request.

[UAE's Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94063)

### 9.) Does your team have all the necessary materials (documents, designs, processes, analytics, APIs, etc) in place for Mobile to make these changes right now?

Yes

### 10.) Has this change been discussed with VAHB POs (Rachel, Ryan, or Don)?

Ryan - Yes, Talking to Don on 8/15

### 11.) Has this change been agreed upon by all impacted VA Stakeholders?

Yes

### 12.) What is your ideal due date for this change request?

Fall 2024

**Comment by timwright12** [comment]
Assigning over to Health & Benefits for review ( @dumathane @ala-yna )

**Comment by benbrasso-agile6** [comment]
@timwright12 is @juancstlm-a6 ok to work on this ticket? What other steps do we need at this point?

**Comment by loripusey** [comment]
@rtwell Will our team need to follow the Collab Cycle process for changes to the native mobile app?

**Comment by timwright12** [comment]
@benbrasso-agile6 yeah, @juancstlm-a6 is basically done all the "good first issue" tickets we have. So, no issues on our end, if he's comfortable in the code he should feel free to work on whatever you all have lined up. we'll always be around for general support as needed and will continue to review PRs

**Comment by benbrasso-agile6** [comment]
Our new sprint cycle starts today 😄

**Comment by rtwell** [comment]
fysa @benbrasso-agile6 @juancstlm-a6 @kristenmcconnell — if your team is doing the work, feel free to take the lead so to speak and start doing things! if you have questions about who's doing what, feel free to reach out and the POs can help clarify.

**Comment by DonMcCaugheyUSDS** [comment]
@rtwell and I reviewed and this is complete and ready to go.  UAE team has everything they need to complete this work independently.  FYI @Hallm13 @kellylein

**Comment by BeccaTupaj** [comment]
@rtwell Can I close this ticket?

**Comment by rtwell** [comment]
I cannot tell if this work is finished, so let's keep it open until we verify @BeccaTupaj

**Comment by dumathane** [comment]
Maybe completed through this pull request? [9859](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/9859)

And I think it spanned these two tickets: [9759](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/9759) and [9762](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/9762)

**Comment by rtwell** [comment]
That question mark makes me less than certain.

**Comment by BeccaTupaj** [comment]
UAE's Epic has been marked as closed and all sub issues are completed: https://github.com/department-of-veterans-affairs/va.gov-team/issues/94063

**Comment by BeccaTupaj** [comment]
@kristenmcconnell Hi there! I am on the new core team and trying to figure out the status of various tickets. Do you know if this work was delivered in December or prior?

**Comment by loripusey** [comment]
Yes, I believe this work is complete. @juancstlm-a6 can you confirm that you implemented these changes and pushed them to production?

**Comment by kristenmcconnell** [comment]
I have this down as releasing to production on November 19, 2024. I see it in demo mode on the app right now.

**Comment by BeccaTupaj** [comment]
Thanks for confirming @kristenmcconnell. I will go ahead and close this ticket. cc: @rtwell
